integration-tests/utils/__init__.py

```diff
@@ -68,3 +71,28 @@ def verify_playbook_execution_status(crc_dispatcher_correlation_id, timeout=30):
                     return True
         time.sleep(5)
     return False
+
+def verify_uploaded_event_runner_data_is_filtered(req_body):
+    """
+    Check that the payload sent by rhc-worker-playbook is filtered down
+    to only the required data.
+    """
```

Comment:
I believe that if you used JSON schema and not yaml schema, then you could use standard `python3-jsonschema` for validation of received data.